cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -55,6 +56,9 @@ func init() {
 
 	Cmd.Flags().BoolVar(&flagNoReport, "no-report", false,
 		"don't report the state")
+
+	Cmd.Flags().BoolVar(&flagSafeMode, "safe-mode", false,
+		"migrate in safe mode")
```

Comment:
we might need more details in the description

Reply:
Added in bcc34f2